app/v1/controllers/DeleteOtherDeductionsController.scala

```diff
@@ -51,12 +55,34 @@ class DeleteOtherDeductionsController @Inject()(val authService: EnrolmentsAuthS
           logger.info(
             s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] - " +
               s"Success response received with CorrelationId: ${serviceResponse.correlationId}")
+
+          auditSubmission(
+            DeductionsAuditDetail(
+              userDetails = request.userDetails,
+              params = Map("nino" -> nino, "taxYear" -> taxYear),
+              requestBody = None,
+              `X-CorrelationId` = serviceResponse.correlationId,
+              auditResponse = AuditResponse(httpStatus = NO_CONTENT, None, None))
```

Comment:
Although it makes no functional difference, I think 
```suggestion
              auditResponse = AuditResponse(httpStatus = NO_CONTENT, response = Right(None))
```
might be clearer/ more consistent with how the other AuditResponse objects are constructed.